modules/rhpam-apb/roles/generate-secret/templates/amq-secret.yml.j2

```diff
@@ -0,0 +1,3 @@
+{{ amq_keystore_name }}: "{{ lookup('file', '{{ amq_keystore_path }}.txt') }}"
```

Comment:
If my premise about the keystore and truststore is correct, this file shouldn't be necessary, otherwise you sould be able to reuse the secret.yml.j2 file with a conditional only for the truststore

Reply:
Actually, to make possible to the users to use it we'll have to add two more information on the secret, client.ts and broker.ts so user can extract the client.ts|ks information from the secret and use on the client side.

Reply:
The user can extract the public key from the keystore directly